Added note to refer users to sgkit (#366)
cggh/scikit-allel: README.rst

.. image:: https://ci.appveyor.com/api/projects/status/1fna2yr9uht0u296?svg=true
    :target: https://ci.appveyor.com/project/alimanfoo/scikit-allel

> Deprecation notice

Reviewer: ```suggestion
Development status
```

Perhaps "deprecation" is a little strong?

Author: Yep, SGTM

******************

Please note that scikit-allel is no longer under active development

> and will soon be in "maintenance-only" mode. The intended successor

Reviewer: ```suggestion
and is now in "maintenance-only" mode. The intended successor
```
